internal/plugins/ansible/v1/init.go

```diff
@@ -105,6 +105,8 @@ func (p *initPlugin) BindFlags(fs *pflag.FlagSet) {
 func (p *initPlugin) InjectConfig(c *config.Config) {
 	c.Layout = pluginKey
 	p.config = c
+	// Just go projects make sense not support multigroup by default
+	p.config.MultiGroup = true
```

Comment:
we are discussing if it should not not be true by default. Anyway, the msgs in https://github.com/operator-framework/operator-sdk/blob/1f2dff149e57704ca7e84c0f24ce74cd723210f1/internal/plugins/helm/v1/scaffolds/api.go#L84-L85 and https://github.com/operator-framework/operator-sdk/blob/1f2dff149e57704ca7e84c0f24ce74cd723210f1/internal/plugins/ansible/v1/scaffolds/api.go#L91-L92 needs to be changed too.